### DISCUSSION TITLE
Blank outputs in frontends

### DISCUSSION THREAD

**Discussion by viridivn** [ggml-org/llama.cpp#7222] [Q&A]
For some reason, the built in web UI works fine, but both SillyTavern and ChatterUI both output blank messages and don't show any output from the model. ChatterUI shows the blank message before processing even finishes on the server side. Does anyone know why this might be?

**Comment by wooooyeahhhh** [comment]
I also had this happen to me. I looked in task manager and there seems to be no utilisation from both cpu and gpu which suggests to me that the api may not be functioning somehow.

**Reply by viridivn** [reply]
For me the CPU does increase, indicating it at least got the request

**Reply by wooooyeahhhh** [reply]
Open an issue I suppose as this bug seems to be caused by llama.cpp not the frontends

**Reply by wooooyeahhhh** [reply]
Created an issue #7231
